homework13/js/main.js

```diff
@@ -0,0 +1,101 @@
+//Завдання 1. У інпут-полі користувач може ввести кількість відповідних секунд появи повідомлення
+
+const inputTime = document.querySelector('.write-number');
+const buttonTimeoutMessage = document.querySelector('.btn-timeout-message');
+
+const messageTimeout = () =>{
+    alert(`russia and russian have left till they disappear.`);
+}
+
+buttonTimeoutMessage.addEventListener('click', (event)=>{
+    event.preventDefault();
+    if(inputTime.value === ''){
+        alert('Поле не може бути пустим!');
+    } else if(inputTime.value.split('-')){
+        alert('Час не може бути від`ємним!');
+    } else{
+        setTimeout(messageTimeout, inputTime.value);
+    }
+})
+
+//Завдання 2.  Відобразити таймер, до дати яку обрав користувач у вигляді
+// "До старту кампанії залишилосьРОКИ:МІСЯЦІ:ГОДИНИ:ХВИЛИНИ"
+
+const buttonTimeoutData = document.querySelector('.btn-timeout-date');
+const inputDate = document.querySelector('.write-date');
+
+function countTime(){
+    const today = new Date();
+    const timerDate = new Date(inputDate.value);
+    let dif = timerDate - today;
+    let years = Math.floor(dif / 31536000000)
+    let month = Math.floor(dif / 2419200000);
```

Comment:
31536000000 and 2419200000 - what are they?

```suggestion
    let years = Math.floor(dif / 31536000000)
    let month = Math.floor(dif / 2419200000);
```